data/json/items/gun/shot.json

```diff
@@ -240,20 +254,53 @@
     "id": "remington_870",
     "copy-from": "shotgun_pump",
     "type": "GUN",
-    "name": "Remington 870",
-    "name_plural": "Remington 870",
-    "description": "One of the most popular shotguns on the market, the Remington 870 is used by hunters and law enforcement agencies alike thanks to its high accuracy and muzzle velocity.",
+    "name": "Remington 870 Wingmaster",
+    "name_plural": "Remington 870 Wingmaster",
+    "description": "With over 10 million made, the Remington 870 is one of the most popular shotguns on the market, and finds use with hunters and law enforcement agencies alike thanks to its high accuracy and muzzle velocity.  This one is a 28 inch barreled model for hunting fowl and game.",
     "weight": "3400 g",
-    "volume": "2500 ml",
+    "volume": "3356 ml",
     "price": 58700,
     "to_hit": -1,
-    "bashing": 12,
+    "bashing": 13,
     "material": [ "steel", "plastic" ],
-    "dispersion": 345,
+    "dispersion": 305,
+    "ranged_damage": 6,
     "durability": 8,
-    "clip_size": 6,
-    "magazines": [ [ "shot", [ "shot_speedloader6" ] ] ],
-    "barrel_length": 3
+    "clip_size": 5,
+    "barrel_length": 17
+  },
+  {
+    "id": "remington_870_breacher",
+    "copy-from": "remington_870",
+    "type": "GUN",
+    "name": "Remington 870 breacher",
+    "description": "This Remington 870 MCS shotgun is setup for breaching operations, with a 10 inch barrel and no stock.  It is small enough to carry as a secondary weapon, specifically to pop open any pesky doors you might come across.  The grip's design makes for controllable yet unpleasant recoil, and the barrel lacks any sights.",
+    "weight": "2812 g",
+    "volume": "1427 ml",
+    "looks_like": "remington_870",
+    "ranged_damage": -5,
+    "skill": "pistol",
```

Comment:
Suppressed shotgun bug
![image](https://user-images.githubusercontent.com/51099123/66661647-5a011700-ec15-11e9-8a0e-e4e723703a4c.png)
Seems it's either suppressed shotgun for muzzle mod or shrinking the gun if you still have the stock mount by swapping to shotgun skill.

Reply:
This is HILARIOUS. ok, the other shotgun pistol, the 12ga pistol has no muzzle location- I guess i'll delete its muzzle here.

Reply:
![image](https://user-images.githubusercontent.com/7764202/66671359-74d98880-ec21-11e9-980c-ff13c02be016.png)

Reply:
Should the breacher really not be able to use a shotgun silencer? The tactical version can apparently take one in real life.
And I dunno if there are guidelines for this somewhere, but why do this (and the winchester 1887) use pistol skill? Just because they are a little shorter?

Reply:
They don't have stocks, and would presumably either be used at the waist or at an arms length. Putting it near your face is a bad idea.

Deleting the muzzle location is the easiest solution I could do to prevent caliber suppressors from being mounted.